src/main/java/services/reqresin/steps/DeleteUserSteps.java

```diff
@@ -8,9 +8,9 @@
 public class DeleteUserSteps {
 
     ReqresService reqresService = new ReqresService();
-    private String DELETE_USER_URL = reqresService.getBaseUri() + ReqresService.POST_USER_URI;
+    private String DELETE_USER_URL = reqresService.getBaseUri() + ReqresService.GET_USERS_URI;
```

Comment:
Like I said in my previous comment, please use a generic name, otherwise this is read as "deleting using the get endpoint". Which is wrong. Use something as USER_URI